tests/utils.go

```diff
@@ -4852,6 +4843,12 @@ func IsRunningOnKindInfraIPv6() bool {
 	return strings.HasPrefix(provider, "kind-k8s-1.17.0-ipv6")
 }
 
+func SkipNetworkPolicyRunningOnKindInfra() {
```

Comment:
Sorry I miss this, we use this only on networkpolicy_test let's move it there, so we don't increment utils.go